source/guides/get-help-from-other-roles.html.md

```diff
@@ -23,10 +24,10 @@ You can use a few pieces of information to help other people understand, assess
 
 ## Product manager and delivery manager
 
-A product manager is responsible for the quality of their products. They use their knowledge of user needs and business goals to frame problems and set priorities for delivery teams. A delivery manager is accountable for the delivery of products and services. You can read more about [product managers' skills](https://ddat-capability-framework.service.gov.uk/product-manager.html) and [deliver managers' skills](https://ddat-capability-framework.service.gov.uk/delivery-manager.html) on the Digital, Data and Technology Profession Capability Framework.
+A product manager is responsible for the quality of their products. They use their knowledge of user needs and business goals to frame problems and set priorities for delivery teams. A delivery manager is accountable for the delivery of products and services. You can read more about [product managers' skills](https://ddat-capability-framework.service.gov.uk/product-manager.html) and [deliver managers' skills](https://ddat-capability-framework.service.gov.uk/delivery-manager.html) on the Digital, Data and Technology (DDaT) Profession Capability Framework.
```

Comment:
```suggestion
A product manager is responsible for the quality of their products. They use their knowledge of user needs and business goals to frame problems and set priorities for delivery teams. A delivery manager is accountable for the delivery of products and services. You can read more about [product managers' skills](https://ddat-capability-framework.service.gov.uk/product-manager.html) and [deliver managers' skills](https://ddat-capability-framework.service.gov.uk/delivery-manager.html) on the Government Digital and Data Profession Capability Framework.
```

DDaT has been rebranded recently.

Reply:
Ah, but the DDaT framework has not been renamed. See https://ddat-capability-framework.service.gov.uk/

I think it should have but they have decided not to do it yet. Not sure why.

Reply:
Yes it has? Just the URL hasn't changed…

![Screenshot 2024-01-17 at 14 16 18](https://github.com/alphagov/design-system-team-docs/assets/121939/2282ff46-a3bd-46c2-b17d-27b2979d06f8)